src/main/java/org/prebid/server/events/EventUtil.java

```diff
@@ -88,10 +81,14 @@ public static void validateAnalytics(RoutingContext context) {
     }
 
     public static void validateTimestamp(RoutingContext context) {
-        final String timestamp = context.request().params().get(TIMESTAMP_PARAMETER);
-        if (StringUtils.isBlank(timestamp)) {
-            throw new IllegalArgumentException(String.format(
-                    "Timestamp '%s' is required query parameter and can't be empty", TIMESTAMP_PARAMETER));
+        final String timestamp = StringUtils.stripToNull(context.request().params().get(TIMESTAMP_PARAMETER));
+        if (timestamp != null) {
+            try {
+                Long.parseLong(timestamp);
```

Comment:
`NumberUtils.isDigits()` or `StringUtils.isDigits()`, (but we still need to check for null bc this value is not required)

Reply:
Both util methods doesn't guaranteed Long range value checking, so `NumberFormatException` can be thrown. Don't think they are useful here.